src/pages/singleLineMap/index.tsx

```diff
@@ -127,75 +125,66 @@ const SingleLineMapPage = () => {
           videoUrl="https://www.youtube-nocookie.com/embed/bXg50_j_hTA?si=inyvqDylStvgNRA6&amp;start=93"
         />
       </Title>
-      <Space direction="vertical" size="middle" style={{ marginBottom: '22px' }}>
-        <Alert message={t('realtime_map_page_description')} type="info" />
```

Comment:
don't we want it?

Reply:
There is a YouTube video for that reason no? Except for that I think the page is pretty self-explanatory

Reply:
you're right, so if this text isn't used anywhere else, perhaps we can remove it from the translation files?

Reply:
Or better - maybe we can add this paragraph to the YouTube modal.
This paragraph, unlike the video, can be translated to English when needed

Reply:
That makes perfect sense I will work on that

Reply:
Thank you! :clap: